frontend/src/components/Sidebar/partials/Content/Content.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { signOut } from 'next-auth/react';
+
+import Flex from '@/components/Primitives/Flex';
+import Separator from '@/components/Primitives/Separator';
+import {
+  ACCOUNT_ROUTE,
+  BOARDS_ROUTE,
+  DASHBOARD_ROUTE,
+  SETTINGS_ROUTE,
+  START_PAGE_ROUTE,
+  TEAMS_ROUTE,
+  USERS_ROUTE,
+} from '@/utils/routes';
+import SidebarItem from '../Item/Item';
+
+export type SidebarContentProps = {
+  strategy: string;
+};
+
+const sidebarItems = [
+  { link: DASHBOARD_ROUTE, iconName: 'dashboard', label: 'Dashboard' },
+  { link: BOARDS_ROUTE, iconName: 'boards', label: 'Boards' },
+  { link: USERS_ROUTE, iconName: 'user', label: 'Users' },
+  { link: TEAMS_ROUTE, iconName: 'team', label: 'Teams' },
+];
+
+const SidebarContent = ({ strategy }: SidebarContentProps) => {
+  const router = useRouter();
+  const [active, setActive] = useState(router.asPath);
+  const isStrategyLocal = strategy === 'local';
+
+  const handleSignOut = async () => {
+    const result = await signOut({
+      callbackUrl: isStrategyLocal ? START_PAGE_ROUTE : '/logoutAzure',
```

Comment:
```suggestion
      callbackUrl: isStrategyLocal ? START_PAGE_ROUTE : AZURE_LOGOUT_ROUTE,
```
This can be extracted to a constant var